/moveit_skill_server:
  ros__parameters:
    planning_pipeline: "ompl"
    planner_id: "RRTstarkConfigDefault"
    planning_attempts: 10
    step_size: 0.005
    min_fraction: 0.50
    max_acceleration_scaling_factor: 0.3
    max_velocity_scaling_factor: 0.1
    planning_time: 1.0
